server/app/controllers/admin/AdminProgramImageController.java

```diff
@@ -106,8 +106,8 @@ public Result updateFileKey(Http.Request request, long programId, String editSta
         request
             .queryString("key")
             .orElseThrow(() -> new IllegalArgumentException("Request must contain file key name"));
-    // TODO(#5676): If Azure support is needed, see ApplicantProgramBlocksController#updateFile for
-    // some additional Azure-specific logic that's needed.
+    // Note: If Azure support is needed, see ApplicantProgramBlocksController#updateFile for
```

Comment:
No plans to add Azure support in the future, so turning this into a note instead of a TODO.